h/api/models/test/document_test.py

```diff
@@ -119,6 +122,416 @@ def test_document_find_or_create_by_uris_no_results():
     assert docuri.type == 'self-claim'
 
 
+create_or_update_document_uri_fixtures = pytest.mark.usefixtures(
+    'DocumentURI',
+    'log',
+)
+
+
+def mock_docuri_dict(uri=None):
+    """Return a mock docuri_dict for create_or_update_document_uri()."""
+    if uri is None:
+        uri = 'http://example.com/example_uri.html'
+
+    now = datetime.datetime.now()
+
+    return {
+        'type': 'self-claim',
+        'claimant': 'http://example.com/example_claimant.html',
+        'uri': uri,
+        'created': now,
+        'updated': now
+    }
+
+
+def mock_document():
+    """Return a mock Document object for create_or_update_document_uri()."""
+    class Document:
+        @property
+        def id(self):
+            pass
+        @property
+        def created(self):
+            pass
+        @property
+        def updated(self):
+            pass
+    return mock.Mock(spec=Document)
```

Comment:
Is there a reason why you couldn't just use the `h.api.models.document.Document` class as a spec for this mock? Then we don't have to make sure to synchronize the API with this class here.